.gitignore

```diff
@@ -7,7 +7,7 @@
 
 #ignore any files in config/, except those in subdirectories.
 /config/*
-!/config/*/*
+!config/**/*/
```

Comment:
My best guess is that Git recently updated the syntax for subfolders, cause it wouldn't work in my PR. You can try to revert it, if the config file changes don't get ignored, you can push the change. Although, in that case, I would wonder why it wouldn't work on my system.